components/rhs_comment/rhs_comment.jsx

```diff
@@ -233,19 +219,13 @@ export default class RhsComment extends React.Component {
         const isSystemMessage = PostUtils.isSystemMessage(post);
         const fromAutoResponder = PostUtils.fromAutoResponder(post);
 
-        let status = this.props.status;
-        if (post.props && post.props.from_webhook === 'true') {
-            status = null;
```

Comment:
For `UserProfile` component - Didn't see the problem while testing it.  I realized the direct effect of it is only to plugin component in case it's using the `status` props.  I added `hideStatus` as suggested.

For `ProfilePicture` - I supposed you specifically mean the one at `PostProfilePicture` which is already being handled.

Reply:
Oh, I see. The `UserProfile` is the user's name, but I thought it included the profile picture as well. Looks good then